tests/signUpValidation.spec.js

```diff
@@ -0,0 +1,211 @@
+            const rePasswordInput = popup.locator('#signupRepeatPassword')
+            const rePasswordRequiredErrorMessage = popup.locator('.invalid-feedback')
+
+            await rePasswordInput.focus()
+            await rePasswordInput.blur()
+            await expect(rePasswordInput).toHaveCSS('border-color', 'rgb(220, 53, 69)')
+            await expect(rePasswordRequiredErrorMessage).toHaveText('Re-enter password required')
+        })
+        test("Re-Password match", async ({page})=>{
+            const inputPasswordString ="qwertY1234"
+            const incorrectRePassword = "qwertY123"
+
+            const popup = page.locator('app-signup-modal')
+            const passwordInput = popup.locator('#signupPassword')
+            const rePasswordInput = popup.locator('#signupRepeatPassword')
+            const passwordMatchErrorMessage = popup.locator('.invalid-feedback')
+
+            await passwordInput.fill(inputPasswordString)
+            await expect(passwordInput).toHaveValue(inputPasswordString)
+
+            await rePasswordInput.fill(incorrectRePassword)
+            await rePasswordInput.blur()
+            await expect(rePasswordInput).toHaveCSS('border-color', 'rgb(220, 53, 69)')
+            await expect(passwordMatchErrorMessage).toHaveText('Passwords do not match')
+        })
+    })
+
+})
+
+test.describe("Sign Up", ()=> {
+    test.beforeEach(async ({page}) => {
+        await page.goto("https://qauto.forstudy.space/")
+        await page.click('text="Sign up"')
+    })
+
+    test("Registration", async ({page})=>{
+        const inputNameString ="Serhii"
+        const inputLastNameString ="Opalinskyi"
+        const inputEmailString ="aqa-sopalinskyi@test.com"
+        const inputPasswordString ="qwertY1234"
+        const inputRePasswordString ="qwertY1234"
+
+        const popup = page.locator('app-signup-modal')
+        const nameInput = popup.locator('#signupName')
+        const lastNameInput = popup.locator('#signupLastName')
+        const emailInput = popup.locator('#signupEmail')
+        const passwordInput = popup.locator('#signupPassword')
+        const rePasswordInput = popup.locator('#signupRepeatPassword')
+
+        await nameInput.fill(inputNameString)
+        await lastNameInput.fill(inputLastNameString)
+        await emailInput.fill(inputEmailString)
+        await passwordInput.fill(inputPasswordString)
+        await rePasswordInput.fill(inputRePasswordString)
+
+        await expect(nameInput).toHaveValue(inputNameString)
+        await expect(lastNameInput).toHaveValue(inputLastNameString)
+        await expect(emailInput).toHaveValue(inputEmailString)
+        await expect(passwordInput).toHaveValue(inputPasswordString)
+        await expect(rePasswordInput).toHaveValue(inputRePasswordString)
+
+        await popup.locator('text="Register"').click()
+        await expect(page).toHaveURL('https://qauto.forstudy.space/panel/garage')
```

Comment:
if your env will be  `https://stage.qauto.forstudy.space/panel/garage`

your test will fail

use relative path without domain